torch/_tensor_docs.py

```diff
@@ -1768,14 +1768,14 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('index_add_',
                r"""
-index_add_(dim, index, tensor, *, alpha=1) -> Tensor
+index_add_(dim, index, input, *, alpha=1) -> Tensor
 
-Accumulate the elements of :attr:`alpha` times :attr:`tensor` into the :attr:`self`
+Accumulate the elements of :attr:`alpha` times :attr:`input` into the :attr:`self`
```

Comment:
This should just be
```
``input``
```
The `:attr:` is responsible for funny business you noted where the attribute links to something

Reply:
Interesting, just curious, is there any reason `:attr:` is used for other args like `alpha`?

Reply:
I don't think there's a good reason for it. I used `:attr:` while writing documentation 3+ years ago for PyTorch because the rest of the docs used `:attr:` but because it does weird things I stopped using it.

Reply:
Oh, interesting. Thanks for the info, I have been a culprit of using it a couple of times in the recent past :D Hence the question. :)